billing/scout-billing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Docker Scout billing
+description: Learn how to buy Docker Scout Team in Docker Hub and manage your subscription
```

Comment:
**suggestion:** why not promote both Scout Team and Scout Business? Otherwise, it might sound like Docker sell Scout Team only.

Reply:
Sure I can add a section that for more than 100 repositories, to contact sales to buy Scout Business.